messybrainz/fetch_artist_mbids.py

```diff
@@ -0,0 +1,117 @@
+# Script to fetch artist MBIDs from MusicBrainz Database using
+# the recording MBIDs in the MessyBrainz database.
+
+import brainzutils.musicbrainz_db.recording as mb_recording
+import json
+import logging
+import re
+
+import messybrainz.default_config as config
+try:
+    import messybrainz.custom_config as config
+except ImportError:
+    pass
+
+from brainzutils import cache
+from brainzutils import musicbrainz_db
+from brainzutils.musicbrainz_db.exceptions import NoDataFoundException
+from messybrainz import db
+from sqlalchemy import text
+
+
+def check_valid_uuid(s):
+    """ Checks if the recording MBID is in the valid format or not.
+        Returns True if its a valid UUID else returns False.
+    """
+    if re.match(r'^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$', s):
+        return True
+    else:
+        return False
+
+
+def add_artist_mbids(recording_mbid, artist_mbids):
+    """ Inserts the artist_mbids corresponding to the recording_mbids
+        into the recording_artist table.
+    """
+    with db.engine.begin() as connection:
+        for artist_mbid in artist_mbids:
+            query = text("""INSERT INTO recording_artist (recording_mbid, artist_mbid)
+                            VALUES (:recording_mbid, :artist_mbid)
+                            RETURNING recording_mbid""")
+            result = connection.execute(query, {
+            "recording_mbid": recording_mbid,
+            "artist_mbid": artist_mbid,
+            })
+
+    return is_recording_mbid_present(recording_mbid)
```

Comment:
This part is strange -- you insert data and then you query for the data again. This means that for every insert there is an unnecessary query to the DB to fetch the data again. Your call to add_artist_mbids() should catch exceptions and add accordingly -- this means that you can trust the INSERT statement to execute correctly and insert the data as you expect.